examples/multion_example_browse.ipynb

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Add some screenshots of the AgentOps dashboard showing the screenshots of a MultiOn session